core/src/main/java/org/mobilitydata/gtfsvalidator/table/AnyTableLoader.java

```diff
@@ -217,4 +252,38 @@ public static GtfsTableContainer loadMissingFile(
         noticeContainer);
     return table;
   }
+
+  private static void parseRowAndColumn(
+      RowParser rowParser,
+      int columnIndex,
+      GtfsTableDescriptor tableDescriptor,
+      GtfsColumnDescriptor columnDescriptor,
+      FieldCache fieldCache,
+      GtfsEntityBuilder builder) {
+    Object value = rowParser.parseValue(columnIndex, columnDescriptor);
+    // If the column isn't present in the input CSV, we don't do anything with the (likely null)
+    // parsed value.
+    if (columnIndex == -1 || value == null) {
+      return;
+    }
+    value = addToCacheIfPresent(value, fieldCache);
+    GtfsSetter<GtfsEntityBuilder, Object> setter =
+        (GtfsSetter<GtfsEntityBuilder, Object>)
+            (useColumnBasedStorage
+                ? columnDescriptor.columnBasedEntityBuilderSetter()
+                : columnDescriptor.entityBuilderSetter());
+    if (useColumnBasedStorage && columnDescriptor.unusedValue()) {
```

Comment:
[question]: To support future extensions that uses _unused_ fields, how can we have a _dynamic_ override of this behavior and load _unused_ columns?